docs/_markbind/navigation/userGuideSections.md

```diff
@@ -5,6 +5,9 @@
   * [Site Configuration]({{baseUrl}}/userGuide/siteConfiguration.html)
 * [Content Authoring]({{baseUrl}}/userGuide/contentAuthoring.html)
   * [Including Contents]({{baseUrl}}/userGuide/includingContents.html)
+  * [Markbind Components]({{baseUrl}}/userGuide/markbindComponents.html)
```

Comment:
`Markbind Components` -> `Using Components`

Remember to make the changes in `header.md` and `site.json` too.